app/shop.py

```diff
@@ -0,0 +1,35 @@
+# from datetime import datetime
+
+
+class Shop:
+    shops = {}
+
+    def __init__(self,
+                 name: str,
+                 location: list,
+                 products: dict) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+        Shop.shops[name] = self
+
+    def products_price(self, product_list: dict) -> float:
+        total = 0
+        for product, amount in product_list.items():
+            total += self.products[product] * amount
+        return total
+
+    def buy_products(self, customer) -> None:
+        # print(f"Date: {datetime.now().strftime('%d/%m/%Y %H:%M:%S')}")
+        # for test we need const time
```

Comment:
We actually don't: there's mocked datetime in test, but you have to import it in a specific way. Import the whole package and use this: 
```python
datetime.datetime.now().strftime("Date: %d/%m/20%y %H:%M:%S\n")